README.md

```diff
@@ -6,19 +6,21 @@
 
 - [About](#about)
 - [GPU device plugin](cmd/gpu_plugin/README.md)
-- [FPGA device plugin](cmd/fpga_plugin/README.md)
-- [FPGA admission controller webhook](cmd/fpga_admissionwebhook/README.md)
-- [FPGA prestart CRI-O hook](cmd/fpga_crihook/README.md)
+- FPGA plugin code
```

Comment:
Is "code" the proper word here? Could be files or repositories or something else?

Reply:
"code" sounds good to me.